examples/python-helm-demo/README.md

```diff
@@ -3,87 +3,168 @@
 
 For this tutorial, we set up Feast with Redis. 
 
-We use the Feast CLI to register and materialize features, and then retrieving via a Feast Python feature server deployed in Kubernetes
+We use the Feast CLI to register and materialize features from the current machine, and then retrieving via a 
+Feast Python feature server deployed in Kubernetes
 
 ## First, let's set up a Redis cluster
 1.  Start minikube (`minikube start`)
-2.  Use helm to install a default Redis cluster
+1.  Use helm to install a default Redis cluster
```

Comment:
Why are these numbers all 1?

Reply:
markdown replaces them with the actual sequence:
https://www.markdownguide.org/basic-syntax/#ordered-lists